pkg/pillar/cmd/zedagent/handlelocation.go

```diff
@@ -59,14 +60,17 @@ func locationTimerTask(ctx *zedagentContext, handleChannel chan interface{}) {
 	for {
 		select {
 		case <-cloudTicker.C:
+			// Loopback the destination, will be fetched on the next iteration
+			triggerCloudLocationInfo <- AllDest
```

Comment:
I'm confused by the naming. What does the "cloud" part of CloudLocationInfo supposed to mean when you are sending this to all destinations? Shouldn't it be named triggerLocationInfo?